engine/hid/src/native/hid_native.cpp

```diff
@@ -228,7 +225,7 @@ namespace dmHID
                 Keyboard* keyboard = &context->m_Keyboards[k];
                 keyboard->m_Connected = 1; // TODO: Actually detect if the keyboard is present
 
-                for (uint32_t i = 0; i < MAX_KEY_COUNT; ++i)
+                for (uint32_t i = 32; i < MAX_KEY_COUNT; ++i)
```

Comment:
Hmm, why we suddenly don't store things in the first integer (m_Keys[0]) ?

Reply:
Eeeh yeah I don't remember actually. I've reverted it